src/layout/footer/Footer.js

```diff
@@ -35,6 +35,8 @@ const intercomShow = () => {
   IntercomAPI('showNewMessage')
 }
 
+const metomicManageCookies = () => window.Metomic('ConsentManager:show')
```

Comment:
This usage of `window` will require a null check, to ensure it exists (e.g. `typeof window !== 'undefined'`).